qiskit/circuit/library/standard_gates/u.py

```diff
@@ -89,13 +89,14 @@ def control(self, num_ctrl_qubits=1, label=None, ctrl_state=None):
             return gate
         return super().control(num_ctrl_qubits=num_ctrl_qubits, label=label, ctrl_state=ctrl_state)
 
-    def _define(self):
-        """Alias for U3 until U becomes a basis gate."""
-        from qiskit.circuit.quantumcircuit import QuantumCircuit
-        q = QuantumRegister(1, 'q')
-        qc = QuantumCircuit(q)
-        qc.u3(self.params[0], self.params[1], self.params[2], q[0])
-        self.definition = qc
+    # def _define(self):
+    #     """Alias for U3 until U becomes a basis gate."""
+    #     from qiskit.circuit.quantumcircuit import QuantumCircuit
+    #     from .u3 import U3Gate
+    #     q = QuantumRegister(1, 'q')
+    #     qc = QuantumCircuit(q)
+    #     qc.append(U3Gate(self.params[0], self.params[1], self.params[2]), [q[0]])
+    #     self.definition = qc
```

Comment:
I might have missed this in an earlier review but is this supposed to be commented out but not removed? Do we need the definition/alias in terms of u3 or not? If not why not just delete this?

Reply:
Deleted in 08383dd 👍